tests/cloud/HL_testing_input.json

```diff
@@ -0,0 +1,17 @@
+{
+    "license": "SEE LICENSE IN LICENSE FILE",
+    "model_name" : ["gpt2","TinyLlama/TinyLlama-1.1B-Chat-v1.0","Salesforce/codegen-350M-mono","wtang06/mpt-125m-c4"],
+    "num_cores" : [8,16],
+    "prompt" : ["My name is"],
+    "prompts_txt_file_path" : ["examples/prompts.txt"],
+    "aic_enable_depth_first" : [0,1],
+    "mos" : [-1,1],
+    "cache_dir" : [null],
+    "hf_token" : [null],
+    "batch_size" : [1,2,4],
+    "prompt_len" : [16,32,64,128],
+    "ctx_len" : [128],
+    "mxfp6" : [0,1],
+    "mxint8" : [0,1],
+    "device_group" : [[0]]
+}

```

Comment:
Why two brackets are used here? device_group is List[int]

Reply:
Because I am taking params as list of values, here device group is itself a list that's why it is list of list.

Reply:
got it.